src/background/index.ts

```diff
@@ -123,7 +123,7 @@ initialize().then((values) => {
   // const httpConnection = new HttpConnection(() => account.getCurrentWallet());
   const memoryCacheRPC = new MemoryCacheRPC();
 
-  new ContentScriptManager().removeExpiredRecords().activate();
```

Comment:
But maybe the intent of keeping this call here was to bring an extra attention to it. To me it just looks natural to keep it in the `activate()`